.github/workflows/Certora.yaml

```
@@ -16,6 +16,7 @@ on:
 
 jobs:
   verify:
+    if: false # certora checks are temporarily disabled
```

actionlint (1.7.11)

[error] constant expression "false" in condition. remove the if: section

(if-cond)